packages/tasit-contracts/test/SimpleStorageWithRemoved.js

```diff
@@ -1,4 +1,4 @@
-const SimpleStorage = artifacts.require("./SimpleStorage.sol");
+const SimpleStorage = artifacts.require("./SimpleStorageWithRemoved.sol");
```

Comment:
Should we rename the `const` or is that overkill?

Reply:
I've initially thought to rename all vars, but to keep standard I've needed to do that on `tasit-action` test cases either and we'll have a lot of big variables. So I've thought that keep vars with the same name as before a better option. What do you think?

Reply:
Works for me!